pinecone/data/grpc/utils.py

```diff
@@ -0,0 +1,107 @@
+import uuid
```

Comment:
Moved these grpc-related utils out of a general `utils.py` file so there wouldn't be any try/except import statements needed in here, and also so they are closer to where they are actually being used in the code.